### PR TITLE
Backporting changes to the doc contribution page

### PR DESCRIPTION
**Pull request by ravage84**
Refers https://github.com/cakephp/docs/pull/2550#issuecomment-87630425

The list of "must haves" starting at https://github.com/cakephp/docs/pull/2573/files#diff-57c701e2395aec4004ed8a648c52070fR35 is probably different for 2.x, though.

### CODE REVIEW

File: en/contributing/documentation.rst [hunk 1]
```diff
@@ -15,7 +20,67 @@ Email the docs team (docs at cakephp dot org) or hop on IRC
...
+If your language is not in the current languages, please contact us through
+Github and we will consider creating a skeleton folder for it. The following
+sections are the first one you should consider translating as these
+files don't change often:
+
+- index.rst
+- intro.rst
+- quickstart.rst
+- installation.rst
+- /intro folder
```
Comment (okinaka) [inline en/contributing/documentation.rst]: I could not find below several files and folder in CakePHP 2.x. Is this only CakePHP 3.0?

Comment (ravage84) [inline en/contributing/documentation.rst]: Yep, this PR was merged too fast.

https://github.com/cakephp/docs/pull/2573#issue-65206365

> The list of "must haves" starting at https://github.com/cakephp/docs/pull/2573/files#diff-57c701e2395aec4004ed8a648c52070fR35 is probably different for 2.x, though.

We should fix that list.

Comment (cake17) [inline en/contributing/documentation.rst]: I can take care of this.
Would this be ok to you ?
- index.rst
- `/cakephp-overview` folder
- `/getting-started` folder
- `/installation` folder
- `/tutorials-and-examples` folder

Comment (ravage84) [inline en/contributing/documentation.rst]: :cool: give it a go!

**Pull request merged**